### PR TITLE
Schedule sms messages sent via the sms-form handler

### PR DESCRIPTION
**Pull request by danielglauser**
In support of Mailing notifications, no card because this is to help with the SMS messages that we are already scheduling. By scheduling them in the future we can update the notifications with the user's mailing preference. By defaulting the delay to zero minutes we won't break when we deploy this and things should behave in a reasonable manner for future API customers.

### CODE REVIEW

File: src/election_notification_works/handlers/sms_form.clj [hunk 1]
```diff
@@ -17,15 +17,20 @@
    :notification-type s/Keyword
    :context {s/Keyword s/Any}
    :medium (s/eq :sms)
+   (s/optional-key :delay-minutes) s/Num
    (s/optional-key :attachments) [{s/Keyword s/Any}]})
-
```
Comment (cap10morgan) [inline src/election_notification_works/handlers/sms_form.clj]: Can we put this blank line separator back in?

File: src/election_notification_works/handlers/sms_form.clj [hunk 2]
```diff
@@ -46,16 +51,21 @@
...
             _ (log/debug "Sending sms-form notification:" (pr-str noty-req)
                          "to user:" (pr-str user))
             response (async/<!!
-                      (notification/send noty-req))]
-        (log/debug "sms-form notification send response:" (pr-str response)
+                      (notification/send-or-schedule noty-req))]
+        (log/debug "sms-form notification send-or-schedule response:" (pr-str response)
                    "for user:" (pr-str user))
+        ;; TODO: We should really return an :error to the front end so it
+        ;; could do something meaninful with it
```
Comment (cap10morgan) [inline src/election_notification_works/handlers/sms_form.clj]: That should be (and is?) already handled by wrapping this in a `handle-errors`. Can you see if that's the case here? If so, we can drop the TODO.

**Comment by cap10morgan** [comment]
ping

**Comment by danielglauser** [comment]
@cap10morgan I'll get back on this today and try to deploy it along with https://github.com/democracyworks/turbovote-web/pull/131

**Comment by danielglauser** [comment]
@cap10morgan Should be good to go. I plan on deploying this first then deploying the changes to `turbovote-web`.

**Comment by cap10morgan** [comment]
LGTM

**Pull request merged**